snap-server.cabal

```diff
@@ -91,7 +91,7 @@ Library
     bytestring                          >= 0.9.1    && < 0.11,
     bytestring-builder                  >= 0.10.4   && < 0.11,
     case-insensitive                    >= 1.1      && < 1.3,
-    clock                               >= 0.4.1    && < 0.6,
+    clock                               >= 0.4.1    && < 0.7,
```

Comment:
not much it seems http://packdeps.haskellers.com/reverse/clock , formatting/tasty/amqp are of popular ones I guess